scripts/HostButton.gd

```diff
@@ -0,0 +1,17 @@
+extends Node
+
+signal set_connect_type
+
+func _ready():
+	print("Setting IP")
+	var myIP
+	for ip in IP.get_local_addresses():
+		if (not ip.begins_with("127")) and ip.count(".") == 3:
+			print(ip)
+			myIP = ip
+	$MyIP.text = "IP: " + str(myIP)
+
+func host():
```

Comment:
See previous comment regarding the naming of this signal handler

Reply:
Fixed.